### PR TITLE
add option to configure options that get shown

### PR DESCRIPTION
**Pull request by jmaslek**
This adds the flag in `chains` that allows user to specify what they want to view.

### CODE REVIEW

File: gamestonk_terminal/options/chains_view.py [hunk 1]
```diff
@@ -124,25 +136,26 @@ def display_chains(symbol: str, expiry: str, other_args: List[str]):
         default=-1,
         help="maximum strike price to consider.",
     )
+    parser.add_argument(
+        "-d",
+        "--display",
+        dest="to_display",
+        default=default_columns,
```
Comment (DidierRLopes) [inline gamestonk_terminal/options/chains_view.py:143]: Instead, I would add here this:
```
type= check_valid_option_chains_headers
```
and define above:
```
def check_valid_option_chains_headers(headers: str) -> str:
    
    columns = [str(item) for item in headers.split(",")]
    
    for header in columns:
        if header not in df_columns:
            raise argparse.ArgumentTypeError("Invalid option chains header selected!")
    
    return columns
```

File: gamestonk_terminal/options/chains_view.py [hunk 2]
```diff
@@ -124,25 +136,26 @@ def display_chains(symbol: str, expiry: str, other_args: List[str]):
...
-    chains_df = chains_df[default_columns].rename(columns=column_map)
-
     try:
         ns_parser = parse_known_args_and_warn(parser, other_args)
         if not ns_parser:
             return
+        if isinstance(ns_parser.to_display, str):
+            columns = list(ns_parser.to_display.split(",")) + ["strike", "option_type"]
+        if isinstance(ns_parser.to_display, list):
+            columns = ns_parser.to_display + ["strike", "option_type"]
```
Comment (DidierRLopes) [inline gamestonk_terminal/options/chains_view.py:156]: This wouldn't be needed, as the ns_parser.to_display would contain the `List[str]` regardless

File: gamestonk_terminal/options/chains_view.py [hunk 3]
```diff
@@ -124,25 +136,26 @@ def display_chains(symbol: str, expiry: str, other_args: List[str]):
         default=-1,
         help="maximum strike price to consider.",
     )
+    parser.add_argument(
+        "-d",
+        "--display",
+        dest="to_display",
+        default=default_columns,
+        help="columns to look at",
```
Comment (DidierRLopes) [inline gamestonk_terminal/options/chains_view.py:144]: In here I would add the columns that the user can select!

File: gamestonk_terminal/options/README.md [hunk 4]
```diff
@@ -84,6 +84,17 @@ Display options chains. [Source: Tradier]
 * --puts : Flag to show puts only
 * -m : Minimum strike price to consider.
 * -M : Maximum strike price to consider.
+* -d       : Columns to display.  Should be comma separated.  Must be in `['bid', 'ask', 'strike', 'bidsize', 'asksize', 'volume', 'open_interest',
```
Comment (DidierRLopes) [inline gamestonk_terminal/options/README.md:87]: Instead, can we have:

```
* -d : Columns to display from the following selection: bid, ask, strike, bidsize, asksize, volume, open_interest, delta, gamma, theta, vega, ask_iv, bid_iv, mid_iv. Default: mid_iv,vega_theta_gamma,delta,volume,open_interest,bid,ask. Note that they are separated by comma.
```

File: gamestonk_terminal/options/README.md [hunk 5]
```diff
@@ -84,6 +84,17 @@ Display options chains. [Source: Tradier]
...
+        'delta', 'gamma', 'theta', 'vega', 'ask_iv', 'bid_iv','mid_iv']`
 
 
 <img width="948" alt="chains" src="https://user-images.githubusercontent.com/25267873/115161876-f708ff80-a097-11eb-8073-195979862a45.png">
+
+To change what columns are displayed, one can use the `-d` flag.  To show the same as above, but only the ask, ask size
```
Comment (DidierRLopes) [inline gamestonk_terminal/options/README.md:93]: I don't think we need this information to follow remaining README.

I think the idea is good, but if we were to add all different options for all different commands, I think these would become quite big and the idea is that the user scrolls through and gets an idea of what he can from the terminal.

File: gamestonk_terminal/options/chains_view.py (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/options/chains_view.py:153]: This makes mypy mad since headers is a List ??

Comment (DidierRLopes) [reply gamestonk_terminal/options/chains_view.py:153]: It should be like this:
```
def check_valid_option_chains_headers(headers: str) -> List[str]:
```

**Review by DidierRLopes — APPROVED** [review]
Perfect!! Thank you!

**Pull request merged**